pkg/service/main.go

```diff
@@ -92,52 +93,67 @@ type TriggerResult struct {
 	Metadata *pb.TriggerMetadata
 }
 
+
+// Now, we don't need the artifact service client in the service layer.
+// However, we keep it here for now because we may need it in the future.
+// service is the implementation of the Service interface
 type service struct {
-	repository               repository.Repository
-	redisClient              *redis.Client
-	temporalClient           client.Client
-	component                *componentstore.Store
-	mgmtPrivateServiceClient mgmtpb.MgmtPrivateServiceClient
-	aclClient                acl.ACLClientInterface
-	converter                Converter
-	minioClient              miniox.MinioI
-	memory                   memory.MemoryStore
-	log                      *zap.Logger
-	workerUID                uuid.UUID
-	retentionHandler         MetadataRetentionHandler
+	repository                   repository.Repository
+	redisClient                  *redis.Client
+	temporalClient               client.Client
+	component                    *componentstore.Store
+	mgmtPrivateServiceClient     mgmtpb.MgmtPrivateServiceClient
+	aclClient                    acl.ACLClientInterface
+	converter                    Converter
+	minioClient                  miniox.MinioI
+	memory                       memory.MemoryStore
+	log                          *zap.Logger
+	workerUID                    uuid.UUID
+	retentionHandler             MetadataRetentionHandler
+	artifactPublicServiceClient  artifactpb.ArtifactPublicServiceClient
+	artifactPrivateServiceClient artifactpb.ArtifactPrivateServiceClient
```

Comment:
❓ Is this one used?

Reply:
Originally, I prepared it because I think service will need to upload and fetch the blob data in the future. 
But, now, I guess we won't need it after we change according to the discussion.


However, to be honest..., I am not sure what is the conclusion in the thread below. I will check it deeper.

Reply:
I will remove it out first. And, in the future, when we need it, let's add it back later.